Constance24/iReporter
# Ft signup page

What does this PR do?
Create a signup page.

Description of tasks to be completed.
Created a signup page that allows a user who visits the page to create a new account.

How should it be manually tested?
Clone the project. Check out the ft-signup-page-#162111281 branch. Open the ui folder and run the signup.html file using a browser.

Relevant pivotal tracker stories.
#162111281
